examples/pruning/mxnet_integration.py

```diff
@@ -0,0 +1,124 @@
+"""
+Optuna example that demonstrates a pruner for MXNet.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+MXNet and MNIST, where the architecture of the neural network and the learning rate of optimizer
+is optimized. Throughout the training of neural networks, a pruner observes intermediate
+results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python mxnet_integration.py
+
+"""
+
```

Comment:
Optuna supports Python2, however, some examples does not do this.
```suggestion
from __future__ import print_function
```